mkdocs.yml

```diff
@@ -7,13 +7,29 @@ repo_url: https://github.com/aquasecurity/trivy
 edit_uri: ""
 
 nav:
-  - HOME: index.md
-  - Getting started:
-      - Overview: getting-started/overview.md
+  - Getting Started: 
+      - Overview: index.md
       - Installation: getting-started/installation.md
       - Quick Start: getting-started/quickstart.md
-      - Further Reading: getting-started/further.md
-  - Docs:
+  - Tutorials:
+      - Overview: tutorials/overview.md
+      - Trivy Kubernetes: 
+          - Trivy K8s: tutorials/kubernetes/trivy-kubernetes.md
+          - GitOps: tutorials/kubernetes/gitops.md       
+      - CI/CD:
```

Comment:
Can we put `CI/CD` on the `Kubernetes` section? CI/CD integration is the first citizen in Trivy.

Reply:
What do you mean?

Reply:
ah got it